content/posts/2022-08-22-okd-on-crc.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Running OKD with CRC"
+date: 2022-08-22T09:14:10+05:30
+---
+
+It has been long time since CRC with an updated OKD bundle was released. Last release was [1.x](https://www.okd.io/crc/#get-started) and
+not updated since then. During our 2.x development cycle, we made effort on CRC's side to decouple the bundles from the release
+artifacts. It allows us to create the bundle independently of the release and upload it to separate location.
+
+Another change we made during 2.x is to add support for presets and from starting we are supporting `podman` and `openshift`
```

Comment:
"from the start, we've been supporting ..." (or "from their introduction, we've been supporting ...")